news/2023/2023-08-19-new-featured-artist-soowamisu.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "New Featured Artist: soowamisu"
+date: 2023-08-19 11:00:00 +0000
+---
+
+A living, breathing collection of breakcore samples (a.k.a. **soowamisu**) is now part of our Featured Artists!
+
+![](https://assets.ppy.sh/artists/376/header.jpg)
+
+**soowamisu** takes an unorthodox approach towards music. Each of their breakcore tunes dives into something new, often disregarding conventional music structure in favour of a distinct aural experience.
+
+**16** of those experiences are available from [**soowamisu**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/376), all pre-timed for your upcoming pristine editor experience.
+
+[COE](/wiki/Community/cavoe's_osu!_event/2023) never ended. Check out [Bubbleman](https://osu.ppy.sh/users/5182050)'s return from retirement as he attempts to play a 4-star **soowamisu** map with touchscreen:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/376/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### soowamisu - .vaporcore
+
+Try out [the map from the video above](https://osu.ppy.sh/beatmapsets/1999116) hosted by [Kyouren](https://osu.ppy.sh/users/2013571)!
+
+<audio controls>
+    <source src="LINK">
+</audio>
+
+### soowamisu - floating architecture of an abstract heaven
+
+Or attempt [this recently Ranked map](https://osu.ppy.sh/beatmapsets/1988496) hosted by [Hinsvar](https://osu.ppy.sh/users/1249323)!
+
+<audio controls>
+    <source src="LINK">
+</audio>
+
+### soowamisu - ethereal cyber grind
+
+<audio controls>
+    <source src="LINK">
+</audio>
+
+### soowamisu - shinjuku
+
+<audio controls>
+    <source src="LINK">
+</audio>
+
+---
+
+Those songs are only the tip of the iceberg. Scroll through [**soowamisu**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/376) to find the rest of the songs (and the template for your next map).
+
+## Featured Artist playlists
+
+In [**YUC'e**'s Featured Artist announcement](https://osu.ppy.sh/home/news/2023-06-03-new-featured-artist-andora#featured-artist-playlists), we announced another batch of [**Featured Artist playlists** in osu!(lazer)](https://osu.ppy.sh/home/news/2023-06-03-new-featured-artist-andora#featured-artist-playlists) for osu! and osu!mania. Now it's time to showcase the prize-winners!
+
+[At the start of last month](https://osu.ppy.sh/home/news/2023-07-15-new-featured-artist-yuce#featured-artist-playlists), we launched a batch of [**Featured Artist playlists** in osu!(lazer)](/wiki/People/Featured_Artists/Featured_Artist_playlists). Those finished just a few days ago, so we're here to announce the results!
```

Comment:
this and the paragraph above say the exact same thing don't they

Reply:
one wasnt supposed to be there